R/Achilles.R

```diff
@@ -651,7 +677,7 @@ achilles <- function (connectionDetails,
   invisible(achillesResults)
 }
 
-#' Create the concept hierarchy
+#' (DEPRECATED) Create the concept hierarchy
```

Comment:
Consider leaving the function implementation since 1) others may still be using it; 2) we should warn users with the deprecate message for at least one version before removing the function later and 3) it is still possible that we may want to use this function, if not for Atlas > Data Source presentation layer purposes, then in other contexts.

Reply:
Hi @t-abdul-basser, good idea. I just restored it with the deprecated warning.